pkg/core/config/main.go

```diff
@@ -45,6 +45,8 @@ var (
 	ErrNotAllowedTemplatedKey = errors.New("not allowed templated key")
 )
 
+const localScmIdentifier = "local"
```

Comment:
This is just a suggestion

```suggestion
const (
    // LOCALSCMIDENTIFIER defines the scm id used to configure the local scm directory  
    LOCALSCMIDENTIFIER string = "local"
    localScmIdentifier = "local"
)
```